### ISSUE TITLE
Inference for AttentionOCR model

### ISSUE THREAD

**Issue by neso613** [tensorflow/models#10293]
Hi there!
I am trying to understand Attention OCR repo and its inference.
I have seen its input/output details - 
![image](https://user-images.githubusercontent.com/31642462/135845091-faaede6e-3d90-4c8e-ac13-97a661a554ce.png)

it says, it takes (32) as string. I don't get this. What does it mean?
In demo_inference.py, there is run method-
def run(checkpoint, batch_size, dataset_name, image_path_pattern):
  images_placeholder, endpoints = create_model(batch_size,dataset_name)
  images_data = load_images(image_path_pattern, batch_size,dataset_name)
  session_creator = monitored_session.ChiefSessionCreator(checkpoint_filename_with_path=checkpoint)
  with monitored_session.MonitoredSession(session_creator=session_creator) as sess:
       predictions = sess.run(endpoints.predicted_text,feed_dict={images_placeholder: images_data})
  return [pr_bytes.decode('utf-8') for pr_bytes in predictions.tolist()]
Here, it takes images_data which --
![image](https://user-images.githubusercontent.com/31642462/135845162-23d1f762-2410-4b25-8d0a-1c80bccdcdfc.png)
as batch_size is 1.

Please help me to know this better.

**Comment by neso613** [comment]
Hi @kumariko 
Any idea for this isuue?

**Comment by neso613** [comment]
@kumariko  This model trained on TF1.x version. Still model can be exported to saved format.
Can we use TF2.x for model and training.
Pls reply.

**Comment by neso613** [comment]
pls update on this.

**Comment by kyscg** [comment]
> @kumariko This model trained on TF1.x version. Still model can be exported to saved format. Can we use TF2.x for model and training. Pls reply.

You should be able to use TF2.x for training and inference

**Comment by neso613** [comment]
> > @kumariko This model trained on TF1.x version. Still model can be exported to saved format. Can we use TF2.x for model and training. Pls reply.
> 
> You should be able to use TF2.x for training and inference

Same issue with TF2.x inferencing. Input details are not clear

**Comment by neso613** [comment]
> @neso613 Could you please try with TF v2.8.0 and let us know if it is still an issue ? Thanks!

sure , I'll do and keep the forum updated 
thanks

**Comment by neso613** [comment]
Hi @sushreebarsa I have upgraded TF version to 2.8 but still no success

![image](https://user-images.githubusercontent.com/31642462/158053582-2192daf8-f318-4aed-a7b3-c826502fdb2d.png)

**Comment by sushreebarsa** [comment]
@neso613 Could you please confirm if you are using the same type of normalization which was used in training?
In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thank you!

**Comment by neso613** [comment]
> @neso613 Could you please confirm if you are using the same type of normalization which was used in training? In order to expedite the trouble-shooting process, please provide a code snippet to reproduce the issue reported here. Thank you!

Yes
